pandas/_libs/groupby_helper.pxi.in

```diff
@@ -879,6 +879,51 @@ def group_cummax_{{name}}(ndarray[{{dest_type2}}, ndim=2] out,
 #----------------------------------------------------------------------
 
 
+@cython.boundscheck(False)
+@cython.wraparound(False)
+def group_any(ndarray[int64_t] out,
```

Comment:
Since these are boolean values they'd probably be better declared as `uint8_t` types, but I was unable to get the function call to work using that type so simply opted for `int64_t` instead